tools/convert/convert.py

```diff
@@ -0,0 +1,82 @@
+import sys
+from argparse import ArgumentParser
+from logging import getLogger
+from os.path import dirname
+
+from rdflib import Graph, Namespace
+from utils import (
+    find_conversions,
+    execute_conversions,
+    standardize_namespaces,
+    get_output_filename,
+)
+
+"""
+Argument Parsing:
+source: source version (for example '1.0.3')
+target: target version (for example '1.1')
+info: log operation description
+"""
+parser = ArgumentParser(description="Update Brick models.")
+parser.add_argument(
+    "models",
+    metavar="model",
+    type=str,
+    nargs="+",
+    help="a turtle file with a brick model",
+)
+parser.add_argument("--source", help="source version", required=True)
+parser.add_argument("--target", help="target version", required=True)
+parser.add_argument(
+    "--info", help="get information related to ongoing operations", action="store_true"
+)
+args = parser.parse_args()
+
+
+# set log level to INFO if required.
+if args.info:
+    getLogger().setLevel("INFO")
+
+
+def convert(source=args.source, target=args.target, models=args.models):
```

Comment:
Why do you define the function in this way instead of just putting the arguments at the end of this script?

Reply:
Sorry, I don't understand. Could you please post a sample snippet/link?

Reply:
Sorry I wasn't clear. Why do you unnecessarily define default values of the arguments here?